checks/http/salesforce.rb

```diff
@@ -0,0 +1,28 @@
+module Intrigue
+module Ident
+module Check
+class SalesForce < Intrigue::Ident::Check::Base
+
+    def generate_checks(url)
+    [
+        {
+            :type => "fingerprint",
+            :category => "application",
+            :tags => ["CRM"],
```

Comment:
Cloud? SaaS? (or is this PaaS?)

Reply:
With this one I wasn't too sure what to class it as since the "ApexPages" is only referenced on their documentation around the Apex programming language.

![image](https://user-images.githubusercontent.com/7880605/78504925-e5440b00-7767-11ea-926c-4368933ef10d.png)

Reply:
I'd call this PaaS